src/test/resources/database/add-category-to-book.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO books_categories (book_id, category_id) values (1, 1);
```

Comment:
Please, stick to the same style in your scripts
```suggestion
INSERT INTO books_categories (book_id, category_id) 
VALUES (1, 1);
```